redirects-www.conf

```diff
@@ -394,33 +397,33 @@ rewrite ^/resources/foia/foiareport2007.pdf /resources/cms-content/documents/foi
 rewrite ^/resources/foia/foiareport2006.pdf /resources/cms-content/documents/foiareport2006.pdf redirect;
 
 # Redirects for /resources/legal-resources/enforcement/
-rewrite ^/resources/legal-resources/enforcement/audits/2016/Friends_of_Erik_Paulsen/5OGC-DFAR1357376.pdf https://www.fec.gov/resources/cms-content/documents/5OGC-DFAR1357376.pdf redirect;
+rewrite ^/resources/legal-resources/enforcement/audits/2016/Friends_of_Erik_Paulsen/5OGC-DFAR1357376.pdf /resources/cms-content/documents/5OGC-DFAR1357376.pdf redirect;
 
 # Redirects for /resources/updates/agendas/2011/
-rewrite ^/resources/updates/agendas/2011/mtgdoc_1145.pdf https://www.fec.gov/resources/cms-content/documents/mtgdoc_1145.pdf redirect;
-rewrite ^/resources/updates/agendas/2011/mtgdoc_1145a.pdf https://www.fec.gov/resources/cms-content/documents/mtgdoc_1145a.pdf redirect;
+rewrite ^/resources/updates/agendas/2011/mtgdoc_1145.pdf /resources/cms-content/documents/mtgdoc_1145.pdf redirect;
+rewrite ^/resources/updates/agendas/2011/mtgdoc_1145a.pdf /resources/cms-content/documents/mtgdoc_1145a.pdf redirect;
 
 # This section is for broader redirects
 rewrite ^/auditsearch/(.*) https://www.fec.gov/legal-resources/enforcement/audit-search/ redirect;
-rewrite ^/audits/([0-9]+)/(.*) https://transition.fec.gov/audits/$1/$2;
+rewrite ^/audits/([0-9]+)/(.*) https://transition.fec.gov/audits/$1/$2; 
 rewrite ^/audio/([0-9]+)/(.*) /resources/audio/$1/$2 redirect;
 rewrite ^/data/advanced/$ /data/browse-data/ redirect;
 rewrite ^/fecletter/(.*) https://www.fec.gov/data/filings/?data_type=processed&form_type=RFAI redirect;
-rewrite ^/finance/(.*) https://transition.fec.gov/finance/$1 redirect;
+rewrite ^/finance/(.*) https://www.fec.gov/data/ redirect?
```

Comment:
@dorothyyeager Just one small typo, replace `?` with `;`

Reply:
Thanks @patphongs - good catch! I've fixed it.